ecl/hql/hqlfold.cpp

```diff
@@ -5582,6 +5582,10 @@ IHqlExpression * CExprFolderTransformer::createTransformed(IHqlExpression * expr
     node_operator op = expr->getOperator();
     switch (op)
     {
+    case no_likely:
```

Comment:
I don't think you want to do this - unless expr->queryChild(0) is a no_constant.  Otherwise you will very quickly lose the likelihood entries from the graph.  The code should also be in doFoldTransformed - so child(0) will have already been constant folded.